src/REPLICA/fix_pimd_langevin.h

```diff
@@ -48,9 +48,9 @@ class FixPIMDLangevin : public Fix {
   double inverse_np;                       // 1.0/np
   double temp;                             // temperature
   double hbar;                             // Planck's constant
-  double lj_epsilon, lj_sigma, lj_mass;    // LJ unit energy, length, and mass scales
-  double other_planck;
-  double other_mvv2e;
+  double lj_epsilon = 1, lj_sigma = 1, lj_mass = 1;    // LJ unit energy, length, and mass scales
```

Comment:
The LAMMPS programming style requests to put initialization of class members into the constructor and not in the header.

Reply:
Sorry my bad. I have just changed this.